### DISCUSSION TITLE
Install from CLI# home/admin/SecurityOnion/*

### DISCUSSION THREAD

**Discussion by BffloSwts** [Security-Onion-Solutions/securityonion#5909] [Unsupported Versions] [answered]
I've tried to install Security Onion from the ISO on ubuntu, after the first couple of screens the install froze so I eventually ended up rebooting.  After logging in the server came up fine with the Ubuntu CLI.  I have a full Security Onion install folder that shows setup, tests, scripts folders etc, however am not sure which install file I need to run to restart the install process.  Any help on how to proceed from CLI # home/admin/SecurityOnion/* would be greatly appreciated.  Maybe my best route is to just nuke the VM and start over, however if it freezes on me again mid install i'd be in the same boat.  
Thanks

**Comment by dougburks** [comment, marked as answer]
If it were me, I'd just go ahead and nuke the VM and start over as that's the quickest and easiest way to ensure a clean installation. For completeness, if you need to re-run Setup, you can follow the steps in the documentation:
https://docs.securityonion.net/en/2.3/installation.html
